src/irisnet/noncore/cutestuff/socks.cpp

```diff
@@ -85,13 +85,13 @@ void SocksUDP::change(const QString &host, int port)
     d->port = port;
 }
 
-void SocksUDP::write(const QByteArray &data) { d->sd->writeDatagram(data, d->routeAddr, d->routePort); }
+void SocksUDP::write(const QByteArray &data) { d->sd->writeDatagram(data, d->routeAddr, quint16(d->routePort)); }
```

Comment:
again. It's better to change type of routePort unless it has some special logic for large number. but very likely there is nothing special.